doc/nrf/ug_multi_image.rst

```diff
@@ -77,32 +77,44 @@ Defining and enabling a child image
 ***********************************
 
 You can enable existing child images in the |NCS| by enabling the respective modules in the parent image and selecting the desired build strategy.
-To turn an application that you have implemented into a child image that can be included in a parent image, you must update the build scripts to make it possible to enable the child image, add the required configuration options, and make sure that image-specific variables can be disambiguated.
+To turn an application that you have implemented into a child image that can be included in a parent image, you must update the build scripts to make it possible to enable the child image and add the required configuration options.
+You should also know how image-specific variables are disambiguated and what targets of the child images are available.
 
 Updating the build scripts
 ==========================
 
 To make it possible to enable a child image from a parent image, you must include the child image in the build script.
+
 This code should be put in place in the cmake tree that is conditional on a configuration option for having the parent image use the child image.
+In the |NCS|, the code is included in the :file:`CMakeLists.txt` file for the samples, and in the MCUboot repository.
```

Comment:
This is an internal implementation detail that they don't need to know.

Reply:
Hmm, not sure I agree. When I'm reading this, I want to know what I need to do to make this work for my child image, and I want to see an example. Putting the code "somewhere in the tree" is very vague, and I think it helps to tell the user where they can look. So it's not meant as implementation detail, but as a pointer to where they can actually see it "in action".

Reply:
I see, that's okay then.